codegen/src/main/ops/org/nd4j/codegen/ops/Image.kt

```diff
@@ -0,0 +1,272 @@
+package org.nd4j.codegen.ops
+
+import org.nd4j.codegen.api.AtLeast
+import org.nd4j.codegen.api.Language
+import org.nd4j.codegen.api.doc.DocScope
+import org.nd4j.codegen.dsl.*
+import org.nd4j.codegen.api.DataType.*
+
+
+
+fun SDImage() =  Namespace("SDImage"){
+    val namespaceJavaPackage = "org.nd4j.linalg.api.ops.custom"
+    Op("CropAndResize") {
+        javaPackage = "org.nd4j.linalg.api.ops.impl.image"
+        javaOpClass = "CropAndResize"
+        Input(NUMERIC, "image") { description = "Input image, with shape [batch, height, width, channels]" }
+        Input(NUMERIC, "cropBoxes") { description = "Float32 crop, shape [numBoxes, 4] with values in range 0 to 1" }
+        Input(NUMERIC, "boxIndices") { description = "Indices: which image (index to dimension 0) the cropBoxes belong to. Rank 1, shape [numBoxes]" }
+        Input(NUMERIC, "cropOutSize") { description = "Output size for the images - int32, rank 1 with values [outHeight, outWidth]" }
+        Arg(NUMERIC, "extrapolationValue") { description = "Used for extrapolation, when applicable. 0.0 should be used for the default" }
+
+        Output(NUMERIC, "output"){ description = "Cropped and resized images" }
+
+        Doc(Language.ANY, DocScope.ALL){
+            """
+ Given an input image and some crop boxes, extract out the image subsets and resize them to the specified size.
+
+ @param image              Input image, with shape [batch, height, width, channels]
+ @param cropBoxes          Float32 crop, shape [numBoxes, 4] with values in range 0 to 1
+ @param boxIndices         Indices: which image (index to dimension 0) the cropBoxes belong to. Rank 1, shape [numBoxes]
+ @param cropOutSize        Output size for the images - int32, rank 1 with values [outHeight, outWidth]
+ @param extrapolationValue Used for extrapolation, when applicable. 0.0 should be used for the default
+ @return Cropped and resized images
+     
+""".trimIndent()
```

Comment:
For each of these, you can remove the old javadoc, now that it's in the input description above.

So the docs section will look like:
```
        Doc(Language.ANY, DocScope.ALL){
            """
            Given an input image and some crop boxes, extract out the image subsets and resize them to the specified size.
            """.trimIndent()
```

Same thing for all the other ops here